apps/arweave/src/ar_chunk_storage.erl

```diff
@@ -52,6 +53,10 @@ put(Offset, Chunk, StoreID) ->
 			Reply
 	end.
 
+store_packed_chunk(Offset, ChunkArgs, StoreID) ->
```

Comment:
The changes to this file have the the ar_data_sync process cast directly to the ar_chunk_storage process covering the same storage module. Previously the ar_chunk_storage listed to the `chunk, packed` ar_event.

Question: is this the correct handling? Previusly *every* ar_chunk_storage process would get notified for every packed chunk - no only the one ar_chunk_storage that covers the chunk's storage_module will be notified.